.spin/cmds.py

```diff
@@ -319,6 +319,52 @@ def _run_asv(cmd):
 
     util.run(cmd, cwd='benchmarks', env=env)
 
+@click.command()
+@click.option(
+    "-b", "--branch",
+    metavar='branch',
+    default="main",
+)
+@click.option(
+    '--uncommitted',
+    is_flag=True,
+    default=False,
+    required=False,
+)
+@click.pass_context
+def lint(ctx, branch, uncommitted):
+    """🔦 Run lint checks on diffs.
+    Provide target branch name or `uncommitted` to check changes before committing:
+
+    \b
+    Examples:
+
+    \b
+    For lint checks of your development brach with `main` or a custom branch:
+
+    \b
+    $ spin lint # defaults to main
+    $ spin lint --branch custom_branch
+
+    \b
+    To check just the uncommitted changes before committing
+
+    \b
+    $ spin lint --uncommitted
+    """
+    click.secho(
+        "Performing lint checks " + "for uncommitted changes" if uncommitted
+        else f"against {branch} branch",
+        bold=True, fg="bright_green",
+    )
```

Comment:
Things work now. Last comment I think: should this be here? Normal behavior of tools is to be silent on success. Echoing what's effectively given by the user on the command line does not seem quite right. WDYT about removing this `secho` completely?

Reply:
Yeah that makes sense, thanks. I'll removed it 👍

Reply:
Done 👍